pkg/minikube/logs/logs.go

```diff
@@ -291,3 +288,19 @@ func logCommands(r cruntime.Manager, bs bootstrapper.Bootstrapper, cfg config.Cl
 
 	return cmds
 }
+
+func enabledAddonPods(cfg config.ClusterConfig) []string {
```

Comment:
add comment, the name is not clear what does it do.

Reply:
Added a comment